api/v2/serializers/details/renewal_strategy.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+
+class RenewalStrategySerializer(serializers.Serializer):
+
+    id = serializers.SerializerMethodField()
+    name = serializers.SerializerMethodField()
+    compute_allowed = serializers.SerializerMethodField()
+    renewed_in_days = serializers.SerializerMethodField()
+    external = serializers.SerializerMethodField()
+
+    def get_name(self,strategy):
+        return strategy[0]
+
+    def get_id(self,strategy):
+        return strategy[1]['id']
+
+    def get_compute_allowed(self,strategy):
+        return strategy[1]['compute_allowed']
+
+    def get_renewed_in_days(self,strategy):
+        return strategy[1]['renewed_in_days']
+
+    def get_external(self, strategy):
+        return strategy[1]['external']
```

Comment:
This isnt the best way to do it , but I was trying to serialize values from a dictionary instead of a model, and the index 1 represents that dict while index 0 is the name of the strategy. This helped me form a defined structure for the rules dict ( used with rules engine )